manifests/params.pp

```diff
@@ -22,7 +22,10 @@
       $virtinst_package = 'virtinst'
       $radvd_package = 'radvd'
       $sysconfig = false
-      $deb_default = {}
+      $deb_default = $::service_provider ? {
+        'systemd' => { 'libvirtd_opts' => '' },  # no '-d', it confuses systemd
```

Comment:
I think I got the issue. I'll assume you are running Debian 7. Until Debian 8 it has a special service provider. On Debian 8 it uses systemd. I did not test on init-style systems but for those the -d seems to be the right way to go.

Are you experiencing the same issue I got with systemd?